doc/admin/deployment.md

```diff
@@ -75,6 +75,69 @@ for latest information and examples.
 At this point of the process, the servers: and provider: section of the yaml
 file can be left blank and will be populated in the subsequent sections.
 
+#### Auto generate configuration file
+
+DAOS can attempt to produce a server configuration file that makes optimal use
+of hardware on a given set of hosts through the 'dmg config generate' command:
+
+```bash
+$ dmg config generate --help
+ERROR: dmg: Usage:
+  dmg [OPTIONS] config generate [generate-OPTIONS]
+
+Application Options:
+...
+  -l, --host-list=  comma separated list of addresses <ipv4addr/hostname:port>
```

Comment:
I think it's worth pointing out that this parameter is optional and that the default behavior is to include all servers in the client configuration's hostlist. I think that this will be the common usecase, as DAOS doesn't really support heterogeneous clusters.

Also, we shouldn't be including the port in example text, so that new users don't think that they have to specify it.

Reply:
don't want to make that change as part of this PR but I will defer to another https://github.com/daos-stack/daos/pull/3820 and update the text here so that admin guide and dmg help text to stay in sync.